README.md

```diff
@@ -138,22 +167,76 @@ gaiacli account <destination_address>
 gaiacli account <your_address>
 ```
 
+You can also check your balance at a given block by using the `--block` flag:
+
+```
+gaiacli account $MYADDR --block=<block_height>
+```
 
-## Staking: Becoming a Validator
+##### Custom fee \(coming soon\)
 
-Get your public key by typing:
+You can also define a custom fee on the transaction by adding the `--fee` flag using the same format:
 
 ```
-gaiad show_validator
+gaiacli send --from=$MYADDR --amount=1000fermion --fee=1fermion --chain-id=<name_of_testnet_chain> --sequence=1 --name=$KEYNAME --to=<destination_address>
+```
+
+### Transfer tokens to other chain
```

Comment:
We aren't running two chains yet and until then we shouldn't have this. Also no one is running a relayer so this will always fail.